src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -23,16 +23,17 @@
  */
 
 /*
- *
- *  (C) Copyright IBM Corp. 1999 All Rights Reserved.
- *  Copyright 1997 The Open Group Research Institute.  All rights reserved.
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 1999, 2023 All Rights Reserved
+ * ===========================================================================
  */
 
 package sun.security.krb5.internal;
 
 import sun.security.krb5.*;
 import sun.security.util.*;
 import java.io.IOException;
+import java.lang.StringBuilder;
```

Comment:
This import is not needed (everything in `java.lang` is imported implicitly).

Reply:
Removed import.